README.md

```diff
@@ -18,6 +18,161 @@ The connector needs the following permissions to the specified bucket:
 
 In case of ``Access Denied`` error see https://aws.amazon.com/premiumsupport/knowledge-center/s3-troubleshoot-403/
 
+### File name format
+
+The connector uses the following format for output files (blobs):
+`<prefix><filename>`.
+
+`<prefix>`is the optional prefix that can be used, for example, for
+subdirectories in the bucket. 
+`<filename>` is the file name. The connector has the configurable
+template for file names. 
```

Comment:
This section is not needed any more as we don't want to promote the prefix thing.

Reply:
I think this is a new one from GCS. Prefix here is "some  subdirectory if you need". So it is not a separate config, it is how filename is created...like in integration tests